dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java

```diff
@@ -63,76 +63,67 @@ public class TelemetryMisconfigTest extends FATServletClient {
+        invalidOtlpEndpointApp = ShrinkWrap.create(WebArchive.class, INVALID_OTLP_ENDPOINT_APP_NAME + ".war")
                         .addPackage(JaxRsMisConfigEndpoints.class.getPackage())
                         .addAsResource(new StringAsset("otel.sdk.disabled=false\notel.traces.exporter=otlp\notel.exporter.otlp.endpoint=" + INVALID_OTLP_ENDPOINT),
                                        "META-INF/microprofile-config.properties");
 
-        ShrinkHelper.exportAppToServer(server, invalidOtlpEndpointApp, SERVER_ONLY);
-
         //Invalid Jaeger timeout amount (Should allow number of milliseconds e.g. 1000)
 
-        WebArchive invalidJaegerTimeoutApp = ShrinkWrap.create(WebArchive.class, INVALID_JAEGER_TIMEOUT_APP_NAME + ".war")
+        invalidJaegerTimeoutApp = ShrinkWrap.create(WebArchive.class, INVALID_JAEGER_TIMEOUT_APP_NAME + ".war")
                         .addPackage(JaxRsMisConfigEndpoints.class.getPackage())
                         .addAsResource(new StringAsset("otel.sdk.disabled=false\notel.traces.exporter=jaeger\notel.exporter.jaeger.timeout=" + INVALID_TIMEOUT),
                                        "META-INF/microprofile-config.properties");
 
-        ShrinkHelper.exportAppToServer(server, invalidJaegerTimeoutApp, SERVER_ONLY);
-
-        //Server name in the endpoint is not known
-        WebArchive notKnownEndpointApp = ShrinkWrap.create(WebArchive.class, NOT_KNOWN_ENDPOINT_APP_NAME + ".war")
+        notKnownEndpointApp = ShrinkWrap.create(WebArchive.class, NOT_KNOWN_ENDPOINT_APP_NAME + ".war")
                         .addPackage(JaxRsMisConfigEndpoints.class.getPackage())
                         .addAsResource(new StringAsset("otel.sdk.disabled=false\notel.traces.exporter=jaeger\notel.exporter.jaeger.endpoint=" + NOT_KNOWN_ENDPOINT),
                                        "META-INF/microprofile-config.properties");
 
-        ShrinkHelper.exportAppToServer(server, notKnownEndpointApp, SERVER_ONLY);
-
-        //The endpoint does not exist
-        WebArchive doesNotExistEndpointApp = ShrinkWrap.create(WebArchive.class, DOES_NOT_EXIST_ENDPOINT_APP_NAME + ".war")
+        doesNotExistEndpointApp = ShrinkWrap.create(WebArchive.class, DOES_NOT_EXIST_ENDPOINT_APP_NAME + ".war")
                         .addPackage(JaxRsMisConfigEndpoints.class.getPackage())
                         .addAsResource(new StringAsset("otel.sdk.disabled=false\notel.traces.exporter=jaeger\notel.exporter.jaeger.endpoint=" + DOES_NOT_EXIST_ENDPOINT),
                                        "META-INF/microprofile-config.properties");
 
-        ShrinkHelper.exportAppToServer(server, doesNotExistEndpointApp, SERVER_ONLY);
-
-        server.startServer();
+        //Don't validate the apps because they have not been deployed yet.
+        server.startServerAndValidate(LibertyServer.DEFAULT_PRE_CLEAN, LibertyServer.DEFAULT_CLEANSTART, false);
```

Comment:
I thought this shouldn't be needed because we haven't deployed the apps yet, but Ben pointed out that they are listed in the server.xml since they use third-party so this might be needed.